R/plots.R

```diff
@@ -125,6 +125,187 @@ plot_pca <- function(obj,
   p
 }
 
+
+#' Plot Loadings and Interpretations 
+#' 
+#' give a principal component, tells you which contribute the most or give a sample, tells you which PC's it contributes to the most
+#' 
+#' @param obj a \code{sleuth} object
+#' @param use_filtered if TRUE, use filtered data. otherwise, use all data
+#' @param sample user input on which sample and which PC's contribute the most
+#' @param PC  principal component to view sample's contribution to that PC
+#' @param units either 'est_counts' or 'tpm'
+#' @param pc_count # of PC's
+#' @param bool scale or not
+#' @param absolute default true, to see all PC's magnitude (recommended)
+#' @return a ggplot object
+#' @export
+plot_loadings <- function(obj, 
+  use_filtered = TRUE,
+  sample = NULL, #string please
+  PC = NULL, #apparently if u type in a string or an integer (corresponding to the PC), they're both ok
```

Comment:
lowercase please. also please remove the comments here and move them up to the roxygen if they are necessary. e.g. 'string required'. also, if you are expecting a particular data type and it will break things otherwise, you should wrap it in a `stopifnot` like online 153